src/render/draw_heatmap.js

```diff
@@ -123,9 +123,10 @@ function bindTextureToFramebuffer(context: Context, painter: Painter, texture: ?
     // Use the higher precision half-float texture where available (producing much smoother looking heatmaps);
     // Otherwise, fall back to a low precision texture
     /* $FlowFixMe[prop-missing] WebGL2 */
-    const type = context.extRenderToTextureHalfFloat ? (context.isWebGL2 ? gl.HALF_FLOAT : context.extTextureHalfFloat.HALF_FLOAT_OES) : gl.UNSIGNED_BYTE;
```

Comment:
Heatmaps work with this fix now with WebGL2, but I'm unsure about removing the `gl.getExtension('EXT_color_buffer_half_float')` check here. /cc @astojilj

Reply:
Unfortunately, this might break some Android GPUs that don't support  `EXT_color_buffer_half_float`. So we're still blocked by [Firefox not supporting `EXT_color_buffer_half_float` in WebGL 2.0 contexts](https://bugzilla.mozilla.org/show_bug.cgi?id=1832535).